<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="GGN000174216001" ignored="4" name="Command line test" tests="6" failures="3" timestamp="25 Dec 2019 15:21:40 GMT" time="6.832" errors="0">
  <testcase name="validateCreateCustomerAPIWithInvalidSecretkey" time="3.442" classname="com.restassured.testcases.CreateCustomerTest">
    <failure type="java.lang.AssertionError" message="expected [200] but found [401]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [401]
at com.restassured.testcases.CreateCustomerTest.validateCreateCustomerAPIWithInvalidSecretkey(CreateCustomerTest.java:38)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
at java.lang.Thread.run(Thread.java:745)
... Removed 15 stack frames]]>
    </failure>
  </testcase> <!-- validateCreateCustomerAPIWithInvalidSecretkey -->
  <testcase name="validateCreateCustomerAPIWithValidSecretkey" time="1.611" classname="com.restassured.testcases.CreateCustomerTest"/>
  <testcase name="validateCreateCustomerAPIWithValidSecretkey" time="1.633" classname="com.restassured.testcases.CreateCustomerTest"/>
  <testcase name="validateCreateCustomerAPIWithValidSecretkey" time="1.637" classname="com.restassured.testcases.CreateCustomerTest"/>
  <testcase name="testData" time="0.0" classname="com.restassured.rough.Testparameterization">
    <failure type="org.testng.TestNGException" message="
Cannot inject @Test annotated Method [testData] with [class java.util.Hashtable].
For more information on native dependency injection please refer to http://testng.org/doc/documentation-main.html#native-dependency-injection">
      <![CDATA[org.testng.TestNGException: 
Cannot inject @Test annotated Method [testData] with [class java.util.Hashtable].
For more information on native dependency injection please refer to http://testng.org/doc/documentation-main.html#native-dependency-injection
... Removed 34 stack frames]]>
    </failure>
  </testcase> <!-- testData -->
  <testcase name="deleteCustomer" time="1.69" classname="com.restassured.testcases.DeleteCustomerTest">
    <failure type="java.lang.AssertionError" message="id is not present expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: id is not present expected [true] but found [false]
at com.restassured.testcases.DeleteCustomerTest.deleteCustomer(DeleteCustomerTest.java:40)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
at java.lang.Thread.run(Thread.java:745)
... Removed 12 stack frames]]>
    </failure>
  </testcase> <!-- deleteCustomer -->
  <testcase name="@BeforeSuite setUp" time="0.0" classname="com.restassured.setup.BaseTest">
    <ignored/>
  </testcase> <!-- @BeforeSuite setUp -->
  <testcase name="@BeforeSuite setUp" time="0.0" classname="com.restassured.setup.BaseTest">
    <ignored/>
  </testcase> <!-- @BeforeSuite setUp -->
  <testcase name="@AfterSuite tearDown" time="0.0" classname="com.restassured.setup.BaseTest">
    <ignored/>
  </testcase> <!-- @AfterSuite tearDown -->
  <testcase name="@AfterSuite tearDown" time="0.0" classname="com.restassured.setup.BaseTest">
    <ignored/>
  </testcase> <!-- @AfterSuite tearDown -->
</testsuite> <!-- Command line test -->
